src/test/ui/existential-type/issue-52843.rs

```diff
@@ -0,0 +1,11 @@
+#![feature(existential_type)]
```

Comment:
Can you also add some comments to this file and rename it to `issue-52843-$description`?